### ISSUE TITLE
Zooming in on a timeline with seconds produces error

### ISSUE THREAD

**Issue by zachwise** [NUKnightLab/TimelineJS3#141]
Reproducible on https://github.com/NUKnightLab/TimelineJS3/blob/master/compiled/examples/wedding.html
example 
`TypeError: undefined is not a function (evaluating 'mask.replace(VCO.Language.DATE_FORMAT_TOKENS, function ($0) {
			return $0 in flags ? flags[$0] : $0.slice(1, $0.length - 1);
		})')`

**Comment by JoeGermuska** [comment]
this traces back to the definition of the date format string for 'millisecond' scale as `1`

https://github.com/NUKnightLab/TimelineJS3/blob/master/source/js/timenav/VCO.TimeAxis.js#L59

What should the date format string for 'millisecond' on the ticks?

**Comment by JoeGermuska** [comment]
@zachwise assign back to me when you have an answer and i'll implement the fix

**Comment by zachwise** [comment]
We should probably do `time_short`for milliseconds ?

**Comment by JoeGermuska** [comment]
So, I set the format for milliseconds to time_sort and then ran into a bug.

When you and I talked f2f I thought it was to do with going beyond the bounds of the list of `zoom_sequences` but even after fixing that, I find that something causes an infinite loop (or other lock up) in the display updating. For the demonstration timeline referenced above, this can be seen to happen on the second click on "zoom in".
